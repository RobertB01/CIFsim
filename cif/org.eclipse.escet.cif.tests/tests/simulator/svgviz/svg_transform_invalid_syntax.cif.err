ERROR: SVG image rendering failed for file "small.svg".
CAUSE: Problem for SVG "rect" element with id "box".
CAUSE: Problem details: attribute.malformed[transform, transform(1,2)]
CAUSE: Problem message: The attribute "transform" of the element <rect> is invalid
