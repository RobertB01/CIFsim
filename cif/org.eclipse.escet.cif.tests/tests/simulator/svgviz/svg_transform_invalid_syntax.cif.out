Initial state: time=0.0, a=*

SVG output ("small.svg") id "box" attr "transform" (SVG attr): "transform(1,2)"

Simulation resulted in deadlock.
